packages/client/components/TeamPrompt/WorkDrawer/LinearIntegrationResults.tsx

```diff
@@ -0,0 +1,164 @@
+import graphql from 'babel-plugin-relay/macro'
+import {PreloadedQuery, usePaginationFragment, usePreloadedQuery} from 'react-relay'
+import {Link} from 'react-router-dom'
+import halloweenRetrospectiveTemplate from '../../../../../static/images/illustrations/halloweenRetrospectiveTemplate.png'
+import {LinearIntegrationResultsPaginationQuery} from '../../../__generated__/LinearIntegrationResultsPaginationQuery.graphql'
+import {LinearIntegrationResultsQuery} from '../../../__generated__/LinearIntegrationResultsQuery.graphql'
+import {
+  LinearIntegrationResults_query$data,
+  LinearIntegrationResults_query$key
+} from '../../../__generated__/LinearIntegrationResults_query.graphql'
+import useLoadNextOnScrollBottom from '../../../hooks/useLoadNextOnScrollBottom'
+import Ellipsis from '../../Ellipsis/Ellipsis'
+import LinearObjectCard from './LinearObjectCard'
+
+type LinearIssueEdge = NonNullable<
+  NonNullable<
+    NonNullable<
+      NonNullable<
+        LinearIntegrationResults_query$data['viewer']['teamMember']
+      >['integrations']['linear']['api']
+    >['query']
+  >['issues']['edges']
+>[number]
+
+// Placeholder type for the issue node, replace with generated type later
+type LinearIssueNode = NonNullable<LinearIssueEdge>['node']
+
+interface Props {
+  queryRef: PreloadedQuery<LinearIntegrationResultsQuery>
+  teamId: string
+}
+
+const LinearIntegrationResults = (props: Props) => {
+  const {queryRef, teamId} = props
+
+  // This query definition must match the one used to generate the queryRef in the parent component (LinearIntegrationResultsRoot)
```

Comment:
-1 if it needs to match, why doesn't it? It only passes some of the arguments
I think the correct way would be to not have a separate query definition in `LinearIntegrationResultsRoot` but instead query this one there.